apache/superset
# fix(helm): remove config overrides for CSRF

### SUMMARY

Fixes #22715

This PR removes the `WTF_` CSRF-related overrides from the configuration file generated by the Helm chart, allowing the defaults from `config.py` to be used.

https://github.com/apache/superset/blob/2ccdb72830ffb549c0112442ba0bc7e4219261d4/superset/config.py#L249-L257

As noted in #22715, there may be better ways to handle the CSRF exclusion for these three routes outside of the config file to prevent this same mistake from being made in other context. This is just the simplest solution that solves the issue with the Helm chart clearing the list.

### TESTING INSTRUCTIONS

1. Deploy a copy of Superset using the Helm chart
2. Navigate around the UI
3. Confirm that calls to `/superset/log` are returning 200 instead of 302 redirects to `/login`

### ADDITIONAL INFORMATION


- [x] Has associated issue: #22715
